orchestration/templates/ingest-processed-archive.yaml

```diff
@@ -1,472 +0,0 @@
-apiVersion: argoproj.io/v1alpha1
```

Comment:
NOTE: The logic from this template is _not_ fully replicated in the new PR. I expect the missing pieces will end up being reimplemented in a different-enough way that having this around wouldn't be very useful.